vault/external_tests/identity/login_mfa_okta_test.go

```diff
@@ -33,48 +69,71 @@ func TestOktaEngineMFA(t *testing.T) {
 
 	_, err = client.Logical().Write("auth/okta/config", map[string]interface{}{
 		"base_url":  "okta.com",
-		"org_name":  org_name,
-		"api_token": api_token,
+		"org_name":  os.Getenv("OKTA_ORG"),
+		"api_token": os.Getenv("OKTA_API_TOKEN"),
 	})
 	if err != nil {
 		t.Fatalf("error configuring okta mount: %v", err)
 	}
 
-	_, err = client.Logical().Write("auth/okta/groups/testgroup", map[string]interface{}{
+	testGroup := os.Getenv("OKTA_TEST_GROUP")
+	if len(testGroup) == 0 {
+		testGroup = "testgroup"
+	}
+
+	_, err = client.Logical().Write("auth/okta/groups/"+testGroup, map[string]interface{}{
 		"policies": "default",
 	})
 	if err != nil {
 		t.Fatalf("error configuring okta group, %v", err)
 	}
 
-	_, err = client.Logical().Write("auth/okta/login/<okta username>", map[string]interface{}{
-		"password": "<okta password>",
+	_, err = client.Logical().Write("auth/okta/login/"+os.Getenv("OKTA_USERNAME"), map[string]interface{}{
+		"password": os.Getenv("OKTA_PASSWORD"),
 	})
 	if err != nil {
-		t.Fatalf("error configuring okta group, %v", err)
+		t.Fatalf("error logging in, %v", err)
 	}
 }
 
 func TestInteg_PolicyMFAOkta(t *testing.T) {
-	t.Skip("This test requires manual intervention and OKTA verify on cellphone is needed")
+	if os.Getenv(logicaltest.TestEnvVar) == "" {
+		t.Skip("This test requires manual intervention and OKTA verify on cellphone is needed")
+	}
+	if !constants.IsEnterprise {
+		t.Skip("PolicyMFA is an enterprise-only feature")
+	}
```

Comment:
I couldn't run this test. It *should* work, but I had to skip this test. Would someone from HashiCorp please test the Policy MFA stuff to make sure this works correctly?